src/components/font_size_editor/font_size_editor.ts

```diff
@@ -31,6 +31,14 @@ css/* scss */ `
     input::-webkit-inner-spin-button {
       -webkit-appearance: none;
     }
+
```

Comment:
Most likely nothing to do with your changes but while we're at it: 
font selector dropdown items don't have `cursor : pointer` + have some horizontal margin visible on hover contrarily to all the other menus.

Reply:
I agree, but the menu should probably have a dynamic width for that, (there is a task somewhere), to not have a menu 200px wide to display a number

Reply:
so, in a next PR